src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -84,7 +84,7 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
                 }
                 catch (TypeLoadException ex)
                 {
-                    if (ex.TypeName == NUnit3FrameworkDriver.CONTROLLER_TYPE)
+                    if (ex.TypeName == "NUnit.Framework.Api.FrameworkController")
```

Comment:
It's temporary, because I didn't want to make CONTROLLER_TYPE public after Rob just made it internal. When I put in the engine, this line will no longer be needed because the Engine itself figures out what kind of exception is involved.